src/kagglesdk/kaggle_http_client.py

```diff
@@ -231,12 +153,7 @@ def _init_session(self):
             return self._session
 
         self._session = requests.Session()
-        self._session.headers.update(
-            {
-                "User-Agent": "kaggle-api/v1.7.0",  # Was: V2
-                "Content-Type": "application/x-www-form-urlencoded",  # Was: /json
-            }
-        )
+        self._session.headers.update({"User-Agent": "kaggle-api/v1.7.0", "Content-Type": "application/json"})  # Was: V2
```

Comment:
I did several changes and did not update this User-Agent either. I guess it doesn't matter as we have not released recently, right? I assume you'll update this the next time you do a release

Reply:
@erdalsivri I thought `kaggle-api/v1.7.0` was the value to use in the next release. It used to be `kaggle-api/v1.0.0`. Should it be something else?